src/admin-views/editor/fieldset/settings-provider.php

```diff
@@ -72,7 +70,9 @@ class="tribe-ticket-editor-field-default_provider settings_field ticket_field"
 						 *
 						 * @param string $module - the name of the module
 						 */
-						echo apply_filters( 'tribe_events_tickets_module_name', esc_html( $module ) );
+						$provider_name = apply_filters( 'tribe_events_tickets_module_name', $active_provider['name'] );
+
+						esc_html_e( $provider_name );
```

Comment:
:no_entry_sign: The $text arg must be a single string literal, not "$provider_name".

<sub>hash: a81a1ffaa27b0813889a3650eb11e898</sub>

Reply:
done